pkgs/top-level/all-packages.nix

```diff
@@ -684,6 +684,9 @@ in
 
   chezmoi = callPackage ../tools/misc/chezmoi { };
 
+  # Without kernel driver, should build and work on non-linux as well
```

Comment:
This comment somewhat disagrees with `meta.platforms` which is restricted to x64-linux & macos (which in turn is inconsistent with the package description which also mentions windows). I think its better to remove this comment & just let `meta.platforms` speak by itself.

Reply:
I made it clearer and explicitly pass no kernel here.